src/gridList/GridListItem.ts

```diff
@@ -274,6 +331,28 @@ export class GridListItem {
         }
     }
 
+    private getWProperty(breakpoint?: string) {
+        const item = this.getItem();
+        const responsiveSizes = item.gridster && item.gridster.options && item.gridster.options.responsiveSizes;
+
+        if (breakpoint && responsiveSizes) {
```

Comment:
`responsiveSizes` is not defined when you drag the prototype item. It has no `gridster` property, therefore the wrong (always 'w' and 'h') size is applied while dragging.

Reply:
Maybe as a first step, we can support prototype items only with `w` and `h`, what do you think?
Otherwise, I'm afraid that bigger changes will be required to make it fully working - `getWProperty` and `getHProperty` would need to have `gridsterOptions` as a required param.